packages/svelte/src/containers/xy-container/xy-container.svelte

```diff
@@ -38,6 +38,8 @@
   let ref: HTMLDivElement
 
   $: chart?.setData(data, true)
+  // eslint-disable-next-line no-self-assign
+  $: config.components = config.components
```

Comment:
Do you know if there are there any drawbacks of doing this, e.g. extra re-renders when nothing changes or something like that?

Reply:
I'm not sure it actually fixes what it was originally supposed to. My original issue was that the chart wasn't rerendering when child component data changes but this doesn't actually fix that. I'm not sure if it's intended that when a child components data is set the whole graph rerenders.

If it is it'll need a different fix and I can look into it lmk :)